Reply/app/src/main/res/layout/nav_drawer_header.xml

```diff
@@ -43,13 +43,14 @@
       app:layout_constraintTop_toTopOf="parent" />
 
   <com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton
-      android:id="@+id/nav_fab"
+      android:id="@+id/nav_drawer_fab"
```

Comment:
Just wondering if the fab id renaming necessary now that you're hiding the drawer from the hierarchy? I guess i'm thinking it could be a nice pattern to keep them with the same id, that way, e.g., you can do a single findViewById(R.id.nav_fab) and set up a single click listener (assuming it can be the same action when either fab is clicked).

Reply:
Ill make a note in the Hand-off doc and tag @hunterstich. This is the current APK in the Screenshot Tests so Ill merge it for now.